src/Microsoft.Health.Dicom.Core/Features/Validation/DicomUidValidation.cs

```diff
@@ -0,0 +1,54 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Text.RegularExpressions;
+using Dicom;
+using Microsoft.Health.Dicom.Core.Exceptions;
+using Microsoft.Health.Dicom.Core.Extensions;
+
+namespace Microsoft.Health.Dicom.Core.Features.Validation
+{
+    internal class DicomUidValidation : DicomElementValidation
+    {
+        private static readonly Regex ValidIdentifierCharactersFormat = new Regex("^[0-9\\.]*$", RegexOptions.Compiled);
```

Comment:
We can end with a period?

Reply:
I copied from https://github.com/microsoft/dicom-server/blob/44acba0557256d7fd3a23894b8c7e0267dbed689/src/Microsoft.Health.Dicom.Core/Features/Validation/DicomElementMinimumValidation.cs#L18.
But you are right. `The UID is a series of numeric components separated by the period "." character.` (http://dicom.nema.org/dicom/2013/output/chtml/part05/sect_6.2.html). I'll fix it.